clients/web/src/views/body/PluginsView.js

```diff
@@ -63,16 +64,30 @@ var PluginsView = View.extend({
             this.failed = _.has(settings, 'failed') ? settings.failed : null;
             this.render();
         } else {
+            const promises = [
+                restRequest({
+                    path: 'system/plugins',
+                    type: 'GET'
+                }),
+                restRequest({
+                    path: 'system/setting',
+                    type: 'GET',
+                    data: {
+                        key: 'core.manage_plugins'
+                    }
+                })
+            ];
+
             // Fetch the plugin list
-            restRequest({
-                path: 'system/plugins',
-                type: 'GET'
-            }).done(_.bind(function (resp) {
-                this.enabled = resp.enabled;
-                this.allPlugins = resp.all;
-                this.failed = _.has(resp, 'failed') ? resp.failed : null;
+            $.when.apply($, promises).done((plugins, managePlugins) => {
+                this.managePlugins = managePlugins[0];
+                this.enabled = plugins[0].enabled;
+                this.allPlugins = plugins[0].all;
+                this.failed = plugins[0].failed;
                 this.render();
-            }, this));
+            }).fail(() => {
+                events.trigger('g:navigateTo', UsersView);
```

Comment:
You've copied code written by [a fool](https://github.com/girder/girder/commit/014e26974c8f5ee7d580207bd3754af62e7b3ac1#diff-a7933f3a28741ea295cda1938cb28217R54)!